dbt/include/bigquery/macros/adapters/apply_grants.sql

```diff
@@ -0,0 +1,29 @@
+{% macro  bigquery__get_show_grant_sql(relation) %}
+    {% if not target.location %}
+        {{ exceptions.raise_compiler_error("In order to use the grants feature, you must specify a location ") }}
+    {% endif %}
+
+    select privilege_type, grantee from {{ relation.project }}.`region-{{ target.location }}`.INFORMATION_SCHEMA.OBJECT_PRIVILEGES
+    where object_schema = "{{ relation.dataset }}" and object_name = "{{ relation.identifier }}" and split(grantee, ':')[offset(1)] != current_user()
+{% endmacro %}
+
+
+{%- macro bigquery__get_grant_sql(relation, grant_config) -%}
+    {%- for privilege in grant_config.keys() -%}
+        {%- set grantees = grant_config[privilege] -%}
+        {%- if grantees -%}
+            {%- for grantee in grantees -%}
+                grant `{{ privilege }}` on {{ relation.type }} {{ relation }} to "{{ grantee }}";
+            {%- endfor -%}
+        {%- endif -%}
+    {%- endfor -%}
+{%- endmacro %}
+
+
+{% macro bigquery__get_revoke_sql(relation, grant_config) %}
+    {%- for privilege in grant_config.keys() -%}
+        {%- for grantee in grant_config[privilege] -%}
+            revoke `{{ privilege }}` on {{ relation.type }} {{ relation }}  from "{{ grantee }}";
```

Comment:
Nit: extra space sneaked in before the `from` clause.

```suggestion
            revoke `{{ privilege }}` on {{ relation.type }} {{ relation }} from "{{ grantee }}";
```

Reply:
IFF BigQuery allows a comma-separated list, then you can copy the examples from Jerco [here](https://github.com/dbt-labs/dbt-core/pull/5452/files/e9899db822c72e16365fb345e8ef53bdf57d8678..16b03333f469189671c47d7127eb9bebdee77d7f) to for both `grant` and `revoke`.